cypress/integration/logistics/materialentnahme_partial_HU.js

```diff
@@ -0,0 +1,244 @@
+      Object.assign(new PackingInstructions(), packingInstructionsJson)
+        .setName(packingInstructionsNameForTU)
+        .apply();
+    });
+  });
+  it('create packing instruction for handling unit - Load unit', function() {
+    cy.fixture('product/packing_instructions.json').then(packingInstructionsJson => {
+      Object.assign(new PackingInstructions(), packingInstructionsJson)
+        .setName(packingInstructionsNameForLU)
+        .apply();
+    });
+  });
+  it('create packing instruction version for handling unit - Transport unit; with packgut - quantity=0.00 and packmittel', function() {
+    cy.fixture('product/packing_instructions_version.json').then(pivJson => {
+      Object.assign(new PackingInstructionsVersion(), pivJson)
+        .setName(packingInstructionsVersionForTU)
+        .setPackingInstructions(packingInstructionsNameForTU)
+        .setPackingMaterial(packingMaterialForTU)
+        .apply();
+    });
+    /**For some reasons,the quantity for the packgut cannot be added in advanced edit mode, only like below */
+    cy.selectTab('M_HU_PI_Item');
+    cy.selectNthRow(1)
+      .find('.Quantity')
+      .dblclick({ force: true })
+      .find('.form-field-Qty input')
+      .type('0', { force: true });
+    cy.selectTab('M_HU_PI_Item');
+  });
+  it('create packing instruction version for handling unit - Loading unit; with packmittel and packingInstructionsNameForTU as unter-packvorschrift', function() {
+    cy.fixture('product/packing_instructions_version.json').then(pivJson => {
+      Object.assign(new PackingInstructionsVersion(), pivJson)
+        .setName(packingInstructionsVersionForLU)
+        .setPackingInstructions(packingInstructionsNameForLU)
+        .setPackingMaterial(packingMaterialForLU)
+        .setUnit('Load/Logistique Unit')
+        .apply();
+    });
+    cy.selectTab('M_HU_PI_Item');
+    cy.pressAddNewButton();
+    cy.selectInListField('ItemType', 'Unter-Packvorschrift', true);
+    /**VERY IMPORTANT!!! the packing instruction for TU have to be set as unter-packvorschrift otherwise the LU quantity won't appear after creating a material receipt */
+    cy.selectInListField('Included_HU_PI_ID', packingInstructionsNameForTU, true);
+    cy.writeIntoStringField('Qty', '10', true, null, true);
+    cy.pressDoneButton();
+  });
+
+  it('Create category', function() {
+    cy.fixture('product/simple_productCategory.json').then(productCategoryJson => {
+      Object.assign(new ProductCategory(), productCategoryJson)
+        .setName(productCategoryName)
+        .apply();
+    });
+  });
+
+  it('Create product for the purchase order', function() {
+    Builder.createProductWithPriceAndCUTUAllocationUsingExistingCategory(
+      productCategoryName,
+      productCategoryName,
+      priceListName,
+      productName1,
+      productName1,
+      productType,
+      packingInstructionsVersionForTU
+    );
+  });
+
+  it('Create vendor', function() {
+    new BPartner({ name: vendorName })
```

Comment:
I think this can be simplified by using the `simple_vendor.json` fixture:
```javascript
cy.fixture('sales/simple_vendor.json').then(vendorJson => {
  new BPartner({ ...vendorJson, name: vendorName })
    .setVendorPricingSystem(priceSystemName)
    .setVendorDiscountSchema(discountSchemaName)
    .apply();
});
```
since the fixture already contains some of this data

Reply:
If we're at this, is it too much if you check all the places where a bpartner (simple_customer, or simple_vendor) are used, and simplify those places as well to use the relevant fixtures and not override the same thing that's already set by the fixture?

In this way, when we copy-pasta next time, we will have the "correct example" wherever we look.

Reply:
done.